src/hooks/usePollMonitor.ts

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import { pollInterval } from "constants/index";
+
+interface UsePollMonitorType {
```

Comment:
I was wondering why use an interface here instead of a type?

Reply:
was wondering this too. i see that it works with an interface, but it looks kinda weird